docs/user-guides/using-tables.md

```diff
@@ -216,3 +216,37 @@ DROP TABLE test_db.test_log_store;
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Details      | Display the table's schema, primary key configuration, partition configuration, properties; as well as the metric information of the files stored in ChangeStore and BaseStore, including the number of files and average file size, as well as the latest submission time of the files. |
+| Files        | Display all partitions and files of the table.                                                                                                                                                                                                                                           |
+| Snapshots    | Display all snapshots of the table, which can be filtered by branch and tag.                                                                                                                                                                                                             |
+| Optimizing   | Display all the self-optimizing processes of the table, each record shows the number and average size of files before and after Optimize, as well as the execution time of each process.                                                                                                 |
+| Operations   | Display the current table's DDL historical change records.                                                                                                                                                                                                                               |
+
+![table-details](../images/admin/table_metrics.png)
+
+![table-optimize-history](../images/admin/table_optimizer_history.png)
+
+## Explore self-optimizing status
+The Optimizing page displays self-optimizing status of all tables.
+![optimizing-metrics](../images/admin/optimizer_metrics.png)
+
+
+- **Optimizing Status**: The current optimizing status of the table, including idle, pending, planning, minor, major, full, committing.
+  - idle: means that self-optimizing is not required on the table.
+  - pending: means that self-optimizing is required on the table and is waiting for resources.
+  - planning: means that self-optimizing process is being planed.
+  - minor: means that minor optimizing is being executed on the table. 
+  - major: means that major optimizing is being executed on the table.
+  - full: means that full optimizing is being executed on the table.
+  - committing: means that self-optimizing process is being committed.
+- **Duration**: The duration of the current status.
+- **File Count**: The total number of files involved in the current Self-optimizing, including base, insert, eq-delete, and pos-delete file types.
```

Comment:
```suggestion
- **File Count**: The total number of files involved in the current self-optimizing, including base, insert, eq-delete, and pos-delete file types.
```

Reply:
Should we describe it in more detail? For example with a self-optimising process?

Reply:
Do you mean that we can add some description about the optimizing process tab in the table detail page here?

If yes, maybe we can add it in another PR.